js/managers/account.js

```diff
@@ -147,6 +147,41 @@ function showInvalidInput(id) {
   var input = getElementById(id);
   input.value = "";
   input.style = "box-shadow: 0 0 10px red; border-color: red;";
+  if (id === "signInUsername") {
```

Comment:
_This is where you connected the two?_ @vanderkyl

Reply:
My question was not a well thought out question. I answered this question for myself. 
*Found this on Mozzillas Developer pages*
_JavaScript has both strict and type–converting comparisons. A strict comparison (e.g., ===) is only true if the operands are of the same type and the contents match._